src/apps/filters/h3.c

```diff
@@ -121,12 +156,19 @@ bool generalHelp(int argc, char *argv[]) {
         .helpText =
             "Convert degrees latitude/longitude coordinate to an H3 cell.",
     };
-    Arg *args[] = {&helpArg, &cellToLatLngArg, &latLngToCellArg};
+    Arg cellToBoundaryArg = {
+        .names = {"cellToBoundary"},
+        .helpText =
+            "Convert an H3 cell to an array of latitude/longitude coordinates "
+            "defining its boundary",
+    };
+    Arg *args[] = {&helpArg, &cellToLatLngArg, &latLngToCellArg,
+                   &cellToBoundaryArg};
     const char *helpText =
         "Please use one of the subcommands listed to perform an H3 "
         "calculation. Use h3 <SUBCOMMAND> --help for details on the usage of "
         "any subcommand.";
-    return parseArgs(argc, argv, 3, args, &helpArg, helpText);
+    return parseArgs(argc, argv, 4, args, &helpArg, helpText);
```

Comment:
The duplication of effort with this help function and needing to increment this little integer on each new verb is kinda annoying. It'd be nice if we could generate this function with a macro or something.

Reply:
@isaacbrodsky any thoughts about potentially generating the entire `generalHelp` function with a macro (that would presumably be fed by other macros the other sub commands use)? Would it be too magical?

Reply:
I'm not sure exactly how you would write such a macro, since you'd need to append to a list, but in principle that sounds OK to me, since the macro magic would be limited to this CLI application.

Reply:
> needing to increment this little integer on each new verb is kinda annoying. It'd be nice if we could generate this function with a macro or something.

I think that you could avoid the manual incrementing of `numArgs` by using `sizeof(args) / sizeof(Arg *)` instead, which could be defined as a macro.